src/plugins/intel_gpu/src/kernel_selector/core/common/kernel_selector_utils.cpp

```diff
@@ -393,13 +403,7 @@ std::vector<size_t> GetOptimalLocalWorkGroupSizes(std::vector<size_t> gws, const
         }
         while (gws[priority_order[i]] % lws_values[lws_idx]) lws_idx++;
 
-        if (lws_max == 256 || total_lws == total_gws) {
-            lws[priority_order[i]] = lws_values[lws_idx];
-        } else {
-            lws[priority_order[i]] = i == 2 && gws[priority_order[0]] != 1 ? 1 : lws_values[lws_idx];
-            if (total_gws > 100 && total_lws < 8 && i == 2)
-                lws[priority_order[i]] = lws_values[lws_idx];
-        }
+        lws[priority_order[i]] = lws_values[lws_idx];
```

Comment:
I'm afraid this might have impact on prev-gen HWs...

Reply:
I add perf drop defense code for dg2. Other platforms have the same behavior as before.